Unix/doc/setup-kerberos-omi.md

```diff
@@ -651,8 +659,29 @@ TBD
 
  - Cause 1: Wrong domain/user/password on command line
  - Cause 2: Expired Credentials
- - Cause 2: Time out of sync with host
- - Cause 3: SSSD/ Winbind setup
+   - you may get the error "kinit: Credentials cache keyring 'persistent:0:0' not found" when running klist command, 
+     just run kinit again will fix this issue, or you can run kinit with long expiration duration: kinit -l 1000h kdcaccount@DOMAINNAME.COM.
+ - Cause 3: Time out of sync with host
+ - Cause 4: SSSD/ Winbind setup
+ - Cause 5: KDC server dns issue
+   - you get the meet error "kinit: Cannot find KDC for requested realm while getting initial credentials" when running klist command, just change "dns_lookup_kdc = false" to "dns_lookup_kdc = true" in /etc/krb5.conf will fix this issue.
+ - Cause 6: Lower domain name
+   - If you get the error "kinit: KDC reply did not match expectations while getting initial credentials", please use upper domain name.
+   - If you get the error "Kerberos log: KDC reply did not match expectations" when enabling Kerberos log:export KRB5_TRACE=/krb5.txt,
+     please use upper domain name in /etc/krb5.conf.
+ - Cause 7: knvo inconsistent issue
+   - you may get the error "Key version number for principal in key table is incorrect" in Kerberos log:export KRB5_TRACE=/krb5.txt, just run these commands to fix this issue:
```

Comment:
Where does the unquote belong in this error message? Is the filename correct, it would be unusual to have a txt file in the root of the filesystem like that. What, **exactly**, is the error message that you're getting here?